src/tree_data.h

```diff
@@ -997,6 +997,20 @@ struct lyd_node_opaq {
     const struct ly_ctx *ctx;       /**< libyang context */
 };
 
+/**
+ * @brief Context term data node record.
+ */
+struct lyd_term_nodes_ext_rec {
```

Comment:
Regarding the renaming, this could then be `lyd_leafref_links_rec`.

Reply:
Done